src/Job/JobQueuePuller.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace AgDevelop\ForkingSupervisor\Job;
+
+class JobQueuePuller implements JobQueuePullerInterface
+{
+    private int $jobId = 1;
+
+    public function __construct(
+        private string $jobClass,
+    ) {
+    }
+
+    public function pull(): JobInterface|null
+    {
+        $job = new $this->jobClass();
+        $job
+            ->setJobId($this->jobId++);
```

Comment:
Czy nie powinniśmy dać tego w jednej linijce?
`$job->setJobId($this->jobId++);` według mnie może tu zostać